sda-commons-server-prometheus/src/main/java/org/sdase/commons/server/prometheus/PrometheusBundle.java

```diff
@@ -107,12 +107,13 @@ private void createPrometheusRegistry(Environment environment) {
                 }));
   }
 
+  @SuppressWarnings("java:S2095")
   private static void bindJvmAndSystemMetricsToGlobalRegistry() {
     // JVM and System Metrics
     new JvmMemoryMetrics().bindTo(Metrics.globalRegistry);
-    try (var jvmGcMetrics = new JvmGcMetrics()) {
-      jvmGcMetrics.bindTo(Metrics.globalRegistry);
-    }
+    // ignore Sonar and not using "try-with-resources" pattern to prevent closing of JVMMetrics
+    // otherwise jvm.gc.pause will not be available
```

Comment:
If there is something closable, it should be closed at the end. There is a [ManagedShutdownListener](https://github.com/SDA-SE/sda-dropwizard-commons/blob/master/sda-commons-server-dropwizard/src/main/java/org/sdase/commons/server/dropwizard/lifecycle/ManagedShutdownListener.java) to register  managed resource that can close on shutdown.

Reply:
@sliesche added that behaviour in [another commit](https://github.com/SDA-SE/sda-dropwizard-commons/commit/3e9fac5bc167d66b3b7e2a9ed94641d0ea196c2c).